tests/LoggerTest.php

```diff
@@ -181,5 +182,11 @@ public function testAdapters(): void
         $logger = new Logger($adapter);
         $response = $logger->addLog($log);
         $this->assertEquals(200, $response);
+
+        // Test Dynatrace
+        $adapter = new Dynatrace(\getenv('TEST_DYNATRACE_KEY').';'.\getenv('TEST_DYNATRACE_ENV_ID'));
+        $logger = new Logger($adapter);
+        $response = $logger->addLog($log);
+        $this->assertEquals(201, $response);
```

Comment:
I ran the test, but it failed. After outputting the response from dynatrace, I got:

```json
{"error":{"code":400,"message":"Could not map JSON at \'properties.stacktrace\' near line 1 column 453"}}
```

Were you able to successfully push an event?

Reply:
@stnguyen90 I don't recall pushing any events to Dynatrace, which is probably why I didn't catch this error. Do you know how we can resolve this issue? I'm willing to help in any way I can.

Reply:
@michizhou, please refer to Dynatrace's docs and see how to properly pass the stacktrace.